hoa-backend/src/auth/auth.module.ts

```diff
@@ -1,8 +1,15 @@
 import { Module } from '@nestjs/common';
 import { AuthController } from './auth.controller';
 import { AuthService } from './auth.service';
+import { JwtModule } from '@nestjs/jwt';
 
 @Module({
+  imports: [
+    JwtModule.register({
+      secret: 'SamMiole',
```

Comment:
:sanaol:

Reply:
I have a question about this actually. What is this really? and how is that related to the secretkey in my strategy class?